components/nav-link.tsx

```diff
@@ -0,0 +1,11 @@
+import Link from 'next/link'
+
+interface NavProps {
+  page: string
+}
+
+export default function NavLink({ page }: NavProps) {
+  const pageName = page.charAt(0).toUpperCase() + page.slice(1)
+
```

Comment:
Nice I like this. Does bake in the assumption that link text will always match the route.

Reply:
Yeah, that's just for now, i intend it to be more dynamic as we go along.  The next link can do plenty on it's own but i like to scaffold a custom one right away for when the app scales.